### PR TITLE
🧱 📝 Audio location video image edit forms

### PR DESCRIPTION
**Pull request by LemmyMwaura**
# Description

Adds the Audio, Location, Video and Image Edit Form. depends on #288, #289 and #294
## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

Screenshot (7)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/audio-input-block-edit/audio-input-block-edit.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="form-wrapper">
+  <form [formGroup]="form" action="" class="form-edit">
+    <span class="title">Audio Input Block</span>
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/audio-input-block-edit/audio-input-block-edit.component.html:3]: Use multi-lang

File: libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/image-input-block-edit/image-input-block-edit.component.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="form-wrapper">
+  <form [formGroup]="form" action="" class="form-edit">
+    <span class="title">Image Input Block</span>
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/image-input-block-edit/image-input-block-edit.component.html:3]: Also here

File: libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/location-input-block-edit/location-input-block-edit.component.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<div class="form-wrapper">
+  <form [formGroup]="form" action="" class="form-edit">
+    <span class="title">Location Input Block</span>
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/location-input-block-edit/location-input-block-edit.component.html:3]: Also here

File: libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/video-input-block-edit/video-input-block-edit.component.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<div class="form-wrapper">
+  <form [formGroup]="form" action="" class="form-edit">
+    <span class="title">Video Input Block</span>
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/video-input-block-edit/video-input-block-edit.component.html:3]: Also here

**Review by ReaganCn — CHANGES_REQUESTED** [review]

File: libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/webhook-edit/webhook-edit.component.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { HttpMethods, HttpMethodTypes, Variable } from '@app/model/convs-mgr/stories/blocks/main';
+import { Observable } from 'rxjs';
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/webhook-edit/webhook-edit.component.ts:4]: order of imports

File: libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/webhook-edit/webhook-edit.component.ts [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    { method: HttpMethodTypes.POST, name: 'POST' },
+    { method: HttpMethodTypes.GET, name: 'GET' },
+    { method: HttpMethodTypes.DELETE, name: 'DELETE' }
+  ];
+
+  variables = new FormControl();
+  variables$: Observable<Variable[]>;
+
+  ngOnInit() {
+    console.log(this.form)
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/webhook-edit/webhook-edit.component.ts:24]: remove

File: libs/features/convs-mgr/stories/blocks/library/video-input-block/src/lib/components/video-input-block/video-input-block.component.scss [hunk 7]
```diff
@@ -1,6 +1,5 @@
 mat-card {
 	min-width: 200px;
-	padding: 10px;
 }
 
 input {
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/stories/blocks/library/video-input-block/src/lib/components/video-input-block/video-input-block.component.scss:5]: code formating

File: libs/features/convs-mgr/stories/blocks/library/webhook-block/src/lib/components/webhook-block/webhook-block.component.html [hunk 8]
```diff
@@ -1,43 +1,6 @@
 <mat-card [id]="id" *ngIf=" webhookForm">
 	<mat-card-content>
 		<form [formGroup]="webhookForm" fxLayout="column" fxLayoutALign="start" fxFlex>
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/stories/blocks/library/webhook-block/src/lib/components/webhook-block/webhook-block.component.html:3]: code formating - 2 spaces

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

File: libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/webhook-edit/webhook-edit.component.ts (hunk 6 above)
Comment (LemmyMwaura) [reply libs/features/convs-mgr/stories/blocks/edit/blocks-edit/src/lib/components/webhook-edit/webhook-edit.component.ts]: fixed

File: libs/features/convs-mgr/stories/blocks/library/video-input-block/src/lib/components/video-input-block/video-input-block.component.scss (hunk 7 above)
Comment (LemmyMwaura) [reply libs/features/convs-mgr/stories/blocks/library/video-input-block/src/lib/components/video-input-block/video-input-block.component.scss:5]: github indentation issue

**Review by ReaganCn — APPROVED** [review]

**Review by ReaganCn — APPROVED** [review]

**Review by ReaganCn — APPROVED** [review]

**Review by IanOdhiambo9 — APPROVED** [review]

**Pull request merged**